draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2009,19 +2157,20 @@ load balancing: this gap has been identified in {{vpn-setup}};
 - a mechanism to report client connectivity constraints imposed by
 some muxponder design: this gap has been identified in {{muxponder}}.
 
-Although not applicable to this document, it has been noted that
-being able to use WSON and Flexi-grid topology models together
-(through multi-inheritance) is not only useful in cases of mixed
-fixed-grid and flexible-grid DWDM network topology but also the only
-viable option in case of a mixed CWDM and DWDM network topology.
+Although not applicable to this document, it has been noted that being
+able to use WSON and Flexi-grid topology models together (through
+multi-inheritance) is not only useful for mixed fixed-grid and
+flexible-grid DWDM network topologies but also the only viable option
+for a mixed CWDM and DWDM network topology.
 
 Although not applicable to this document, it has been noted that the
-WDM tunnel model would support also optical tunnel setup in case of a
-mixed CWDM and DWDM network topology.
+DWDM tunnel model would also support optical tunnel setup in the case
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
WDM tunnel model would also support optical tunnel setup in the case
```